<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SwitchScriptClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="layoutMain">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabScript">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>脚本</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutScript">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="textEditScript"/>
          <widget class="QTextEdit" name="textEditLog"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSignal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>信号</string>
       </attribute>
       <layout class="QHBoxLayout" name="layoutSignal" stretch="300,115">
        <item>
         <widget class="QVideoWidget" name="videoPlayer" native="true"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupSignalSetting">
          <property name="title">
           <string>设置</string>
          </property>
          <layout class="QVBoxLayout" name="layoutSignalSetting">
           <item>
            <layout class="QHBoxLayout" name="layoutVideoInputDevice" stretch="5,9">
             <item>
              <widget class="QLabel" name="labelVideoInputDevice">
               <property name="text">
                <string>采集设备：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboVideoInputDevice"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layoutSampleRate" stretch="5,9">
             <item>
              <widget class="QLabel" name="labelSampleRate">
               <property name="text">
                <string>采样率：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinSampleRate"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelSignalList">
             <property name="text">
              <string>信号列表：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="signalList">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layoutSignalOperation">
             <item>
              <widget class="QPushButton" name="buttonAddSignal">
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonRemoveSignal">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="menuActionNew"/>
    <addaction name="menuActionOpen"/>
    <addaction name="menuActionSave"/>
    <addaction name="menuActionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>运行(&amp;R)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolBarActionRun"/>
   <addaction name="toolbarActionStop"/>
  </widget>
  <action name="menuActionOpen">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="menuActionSave">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="menuActionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="menuActionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="toolBarActionRun">
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="toolbarActionStop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QtMultimediaWidgets/QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
